docs/installation-guide-azure.md

```diff
@@ -81,6 +81,14 @@ export ARM_ENVIRONMENT="german"
 export TF_VAR_azure_cloud=AZUREGERMANCLOUD
 ```
 
+Optionally for VPN support add following variables. `bastion_cidr` should be unique and a part of `vnet_cidr` (10.0.0.0/16 by default). Recommended to use /28 subnets from range 10.0.4.0/22 (e.g. 10.0.4.0/28, 10.0.4.16/28, etc.).
```

Comment:
Shouldn't we reference this page here ? https://github.com/giantswarm/giantswarm/wiki/Giant-Swarm-VPN 
Bastion cidr should be unique across that table

Reply:
I'm trying to avoid referencing any internal repos / wiki , because it's open source repo.

Reply:
Ah, ok